.github/workflows/publish-binaries.yml

```diff
@@ -76,13 +76,13 @@ jobs:
           cd build
           cp $(node -e "console.log(process.execPath)") ${{ matrix.settings.system != 'windows' && 'node' || 'node.exe' }}
           npm init -y
-          npm install ironfish
+          npm install ironfish@${{ github.event.release.tag_name }}
```

Comment:
Will this be `ironfish@v1.20.0` instead of `ironfish@1.20.0`?

Reply:
Also if the command didn't work previously because the latest version wasn't published yet would it even be able to find that latest version? Thinking maybe we could fail the job if the latest version on npm doesn't match the tag. `npm view ironfish version` would get latest version looks like

Reply:
Both are valid, surprisingly. I was expecting to have to do some string manipulation, but it ended up working just fine. https://github.com/mat-if/foo/blob/main/.github/workflows/foo.yml#L25